ui/src/components/Plot1D.jsx

```diff
@@ -3,7 +3,6 @@ import { connect } from 'react-redux';
 import Dygraph from 'dygraphs';
 import 'dygraphs/dist/dygraph.min.css';
 
-// eslint-disable-next-line react/no-unsafe
```

Comment:
Worth noting that ESLint is now much better at detecting misused inline directives.